politeiawww/events.go

```diff
@@ -535,11 +535,7 @@ func (e *EventManager) _register(eventType EventT, listenerToAdd chan interface{
 		e.Listeners = make(map[EventT][]chan interface{})
 	}
 
-	if _, ok := e.Listeners[eventType]; ok {
-		e.Listeners[eventType] = append(e.Listeners[eventType], listenerToAdd)
-	} else {
-		e.Listeners[eventType] = []chan interface{}{listenerToAdd}
-	}
+	e.Listeners[eventType] = append(e.Listeners[eventType], listenerToAdd)
```

Comment:
gofmt linter was requesting this code change: https://staticcheck.io/docs/checks#S1036